rustls/src/client/mod.rs

```diff
@@ -603,3 +610,33 @@ pub trait ClientQuicExt {
 
 #[cfg(feature = "quic")]
 impl ClientQuicExt for ClientConnection {}
+
+#[test]
```

Comment:
Since we don't have tests in this module and this is testing the public API, maybe just add this test in `tests/api.rs`, like most of the others? (And if it stays here it should go in a `#[cfg(test)] mod tests {}` block.)